.github/workflows/release.yaml

```diff
@@ -0,0 +1,123 @@
+name: release
+on:
+  pull_request:
+    paths:
+    - '.github/workflows/release.yaml'
+    - 'dist/*'
+    - 'tools/*'
+  push:
+    branches:
+    - main
+    tags:
+    - "*"
+
+jobs:
+  package:
+    runs-on: ubuntu-latest
+    env:
+      RUST_VERSION: "1.57.0"
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+        - name: debian
+          version: 11
+          codename: bullseye
+          pkg_type: deb
+        - name: ubuntu
+          version: 21.04
+          codename: hirsute
+          pkg_type: deb
+        - name: centos
+          version: 7
+          codename: 7
+          pkg_type: rpm
+        - name: centos
+          version: 8
+          codename: 8
+          pkg_type: rpm
+        postgres:
+        - version: "12"
+        - version: "13"
+        - version: "14"
+
+    steps:
+    - name: Install PackageCloud
+      run: |
+        sudo gem install package_cloud --no-doc
+    - uses: actions/checkout@v2
+    - name: Gather Metadata
+      id: metadata
+      run: |
+        version="nightly";
+        if [ "${GITHUB_REF}" =~ 'refs/tags' ]; then
+          version="$(echo "${GITHUB_REF}" | sed -e 's|refs/tags/||')";
+        fi;
+        echo "::set-output version=${version}"
+        echo "::set-output filename=promscale_extension-${version}.${{ matrix.os.name }}${{ matrix.os.version }}.amd64.${{ matrix.os.pkg_type }}"
+        echo "::set-output image=promscale_extension:${version}.${{ matrix.os.name }}${{ matrix.os.version }}.amd64.${{ matrix.os.pkg_type }}"
+
+    - name: Setup QEMU
+      uses: docker/setup-qemu-action@v1
+
+    - name: Setup Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Build Package
+      uses: docker/build-push-action@v2
+      with:
```

Comment:
Yeah for sure! I plan to enable that (that's why I added the qemu action), but wanted to see how long a full run of the workflow takes before doing so, in case it's pushing the limits for a single workflow run. If we can do it all in a single workflow run, that would be ideal. If it takes too long to build all of the packages in a single run, then I'd probably do all of the amd64 packages in the primary workflow, and then trigger a second run on completion of the first that appends the arm64 packages to the release that gets created.

I'll make sure that gets addressed by the time this is merged though!